### PR TITLE
Localize toast menu and steptool

### PR DESCRIPTION
**Pull request by kanaxx**
Translated some words into Japanese
- notification toast config in app config
- step tool in image editor

Could you please check and merge them?

### CODE REVIEW

**Comment by Jaex** [comment]
English ContentAlignment texts not correct. before it was like this:

File: ShareX.HelpersLib/Properties/Resources.ja-JP.resx [hunk 1]
```diff
@@ -1213,4 +1213,83 @@
   <data name="BorderStyle_DashDotDot" xml:space="preserve">
     <value>二点鎖線</value>
   </data>
+  <data name="ToastClickAction_CloseNotification" xml:space="preserve">
+    <value>通知を閉じる</value>
+  </data>
+  <data name="ToastClickAction_AnnotateImage" xml:space="preserve">
+    <value>画像編集</value>
+  </data>
+
```
Comment (Jaex) [inline ShareX.HelpersLib/Properties/Resources.ja-JP.resx:1222]: Why empty line here?

**Comment by Jaex** [comment]
Thanks!

**Comment by kanaxx** [comment]
Thanks for checking my sources.

**Pull request merged**